Dockerfile

```
@@ -1,157 +1,133 @@
-      [ -f "$source_dir/$caps_file" ] && cp "$source_dir/$caps_file" "$dest_dir/${ext_name}.capabilities.json"; \
-      echo "  -> $dest_dir/${ext_name}.wasm"; \
-    done; \
-    count=$(find /app/wasm-bundles -name '*.wasm' | wc -l); \
-    echo "Built $count WASM extensions"; \
-    [ "$count" -gt 0 ] || { echo "ERROR: No WASM extensions were built"; exit 1; }
-
-# Stage 5a: Shared runtime base
-FROM debian:bookworm-slim AS runtime-base
+FROM debian:bookworm-slim AS runtime
 
 RUN apt-get -o Acquire::Retries=3 update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)